package.json

```diff
@@ -32,7 +32,6 @@
     "@storybook/addon-actions": "^3.4.3",
     "@storybook/addon-links": "^3.4.3",
     "@storybook/vue": "^3.4.3",
-    "@vue/composition-api": "^1.4.1",
```

Comment:
this one is ok to be in devDeps and is required, as peerDeeps are working only when package is used as a part of another solution.

I'll handle it.